### PR TITLE
Initial load of Node + Postgres post.

### PR DESCRIPTION
**Pull request by leebrandt**

### CODE REVIEW

File: _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md [hunk 1]
```diff
@@ -0,0 +1,729 @@
+---
+layout: blog_post
+title: "Build a REST API with Node and Postgres"
```
Comment (mraible) [inline _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md]: Should this be PostgreSQL?

File: _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md [hunk 2]
```diff
@@ -0,0 +1,729 @@
...
+If you don't already have one, [sign up for a forever-free developer account](https://developer.okta.com/signup/). Log into your developer console, navigate to **Applications**, then click **Add Application**. Select **Web**, then click **Next**.
+
+You'll want to change the default ports from `8080` to `3000` to match your server. Your settings should then look something like this:
+
+{% img blog/node-postgres/okta-application-settings.png alt:"New Application Settings" width:"800" %}{: .center-image }
+
+Click **Done** to save your app, then copy your **Client ID** and **Client Secret** and paste them as variables into your `.env` file in the root of your project. You'll also need to add your organization URL (without the `-admin` suffix). Add these three variables to your existing `.env` file:
+
+```bash
+OKTA_ORG_URL=https://{yourOktaOrgUrl}
```
Comment (mraible) [inline _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md]: Should be yourOktaDomain

File: _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md [hunk 3]
```diff
@@ -0,0 +1,729 @@
...
+$ curl -sH "$(cat headers.txt)" http://localhost:3000/my-titles | json -o inspect
+[ { id: 3,
+    location: null,
+    title: { id: 'tt0944947', name: 'Game of Thrones (2011–)' },
+    service: { id: 4, name: 'HBO Go' } } ]
+```
+
+## Node + Postgres API Takeaways
+
+That's it! You should now have a fully authenticated API to keep track of your movies and TV shows that uses Postgres to store data and Okta + OAuth2 for API authentication. I hope you enjoyed working on this little project with Node and Postgres. If you want to see the final code sample for reference, you can find it [on GitHub](https://github.com/oktadeveloper/okta-node-postgres-example).
```
Comment (mraible) [inline _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md]: Repo needs title added (at the top) and link to blog post.

**Comment by mraible** [comment]
Minor issue about `yourOktaDomain`

File: _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md (hunk 3 above)
Comment (leebrandt) [reply _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md]: Can't make link to blog post until it's live...

File: _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md (hunk 1 above)
Comment (leebrandt) [reply _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md]: Nope: Keywords are Node and Postgres

File: _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md (hunk 3 above)
Comment (mraible) [reply _source/_posts/2019-03-27-build-rest-api-with-node-and-postgres.md]: Sure you can. It'll be https://developer.okta.com/blog/2019/03/27/build-rest-api-with-node-and-postgres.

**Pull request merged**